src/renderer/components/ConfigureServer.tsx

```diff
@@ -35,8 +35,9 @@ type ConfigureServerProps = {
     onConnect: (data: UniqueServer) => void;
 };
 
+let validationTimeout: NodeJS.Timeout;
```

Comment:
Any reason to tie this timeout to the file instead of to the component (using a `useRef`)?